server/coordinator.go

```diff
@@ -452,7 +482,7 @@ func (c *coordinator) runScheduler(s *scheduleController) {
 		select {
 		case <-timer.C:
 			timer.Reset(s.GetInterval())
-			if !s.AllowSchedule() {
+			if !s.AllowSchedule() || s.isPaused() {
```

Comment:
How do you think about the idea that put `s.isPaused` inside of `s.AllowSchedule`? When a scheduler is paused, I think it means that the scheduler is not allowed to do the scheduling.

Reply:
Finish. sorry that trouble you to review so many times...